File: 240624/특정 문자로 시작하는 문자열/strings-that-start-with-a-specific-character.py
n=int(input())
arr=[
    input() for _ in range(n)
]
a=input()
cnt=0
len_sum=0
for i in arr:
    if i[0]==a:
        cnt+=1
        len_sum+=len(i)
print(f"{cnt} {len_sum/cnt:.2f}")

File: 250624/특정 문자로 시작하는 문자열/strings-that-start-with-a-specific-character.cpp
#include <iostream>
#include <string>
using namespace std;

int main() {
    // Please write your code here.
    int n;
    cin >> n;
    int cnt=0;
    int sum=0;
    string arr[20];
    char c;
    for (int i=0; i<n; i++) {
        cin >> arr[i];
    }
    cin >> c;
    for (auto i : arr) {
        if (i[0] == c) {
            cnt+=1;
            sum+=i.length();
        }
    }
    cout << fixed;
    cout.precision(2);
    cout << cnt << " " << (double)sum/cnt;
    return 0;
}